mssql_log_collector/cron-mysql-log.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+AGENT="mssql-log"
```

Comment:
rename this file to 'cron-mssql-log.sh' to reflect the Agent type.

Reply:
renamed